runhouse/servers/http/http_client.py

```diff
@@ -698,3 +699,20 @@ def keys(self, env=None):
         return self.request(
             f"keys/?env_name={env_name}" if env_name else "keys", req_type="get"
         )
+
+    def list_processes(self):
+        return self.request_json("/processes", req_type="get")
+
+    def create_process(
+        self,
+        name: str,
+        compute: Optional[Dict] = {},
```

Comment:
small nit - can't we set these as `None` by default instead and the base model will handle appropriately? setting them as mutable feels wrong